core-libs/setup/jest.config.js

```diff
@@ -20,4 +20,9 @@ module.exports = {
       lines: 90,
     },
   },
+  //Will become default in v29, and can be removed. See: https://jestjs.io/docs/upgrading-to-jest29
+  snapshotFormat: {
```

Comment:
We had to downgrade Jest to version 28 because `jest-preset-angular` in version 12.2.3 (which is the latest btw.) doesn't support Jest 29.
Due to different formatting of snapshot files between v28 and v29, we decided to set formatting that is the default for v29 to not re-generate snapshots twice - now and in the future after bumping the version pack to 29 as soon as `jest-preset-angular` will support it.
This change has been consulted with @znikola